docs/howto/system_testing.md

```diff
@@ -66,7 +79,42 @@ services:
 
 Here, `SERVICE_LOGS_DIR` is a special keyword. It is something that we will need later.
 
-### Data stream-level configuration
+### Terraform service deployer
+
+The `*.tf` files define the infrastructure using the Terraform syntax. The terraform based service can be handy to boot up
+resources on AWS and use them for testing (e.g. spawn EC2 instance and collect its metrics).
```

Comment:
Should we specifically mention AWS here? Or can we be more generic and say resources supported by Terraform or something like that?

Reply:
Fixed